servicetalk-benchmarks/build.gradle

```diff
@@ -33,9 +32,9 @@ dependencies {
   implementation project(":servicetalk-transport-netty-internal")
   implementation project(":servicetalk-loadbalancer")
   implementation project(":servicetalk-data-jackson")
-  implementation "com.google.code.findbugs:jsr305"
+  implementation "com.google.code.findbugs:jsr305:$jsr305Version"
   implementation "io.netty:netty-codec-http"
```

Comment:
since there is only one netty dependency, would it make sense to also just specify it with $nettyVersion and not pull in the bom for consistency?

Reply:
when a library publishes a bom, it's always best to use bom even for 1 dependency because it helps build tools to align version between other modules when final applications brings more modules

Reply:
plus netty-bom helps to bring correct tcnative dependency, if we manage it manually we will always have to check what tcnative version is compatible with the current netty version

Reply:
ok that's a good second point, I'll take it 😅